inc/Image_Sizes/Component.php

```diff
@@ -51,7 +50,8 @@ public function filter_content_image_sizes_attr( string $sizes, array $size ) :
 			$sizes = '100vw';
 		}
 
-		if ( wp_rig()->is_primary_sidebar_active() ) {
+		if ( method_exists( 'wp_rig', 'is_primary_sidebar_active' )
```

Comment:
I think this should be `method_exists( wp_rig(), 'is_primary_sidebar_active' )` right?

Reply:
It seems I have not tested my code properly: it doesn't work 🤦‍♂  
`method_exists( wp_rig(), 'is_primary_sidebar_active' )` doesn't work either.
`method_exists( 'WP_Rig\WP_Rig\Sidebars\Component', 'is_primary_sidebar_active' )` works, but only if the component has been completely removed (doesn't work if the component is simply not instanciated). I'll need some help to find the right solution.